dockstore-webservice/src/main/java/io/dockstore/webservice/core/Workflow.java

```diff
@@ -161,6 +162,10 @@ public abstract class Workflow extends Entry<Workflow, WorkflowVersion> {
     @JoinColumn(name = "actualDefaultVersion", referencedColumnName = "id", unique = true)
     private WorkflowVersion actualDefaultVersion;
 
+    @Column(nullable = true)
+    @Schema(type = "integer", format = "int64", description = "The timestamp of the most recent version control release, such as a GitHub release")
+    private Timestamp latestReleaseDate;
```

Comment:
I put this in Workflow instead of Entry because the only tools we support in the .dockstore.yml are AppTools. But I could see an argument to put it in Entry.